src/blueprint/azext_blueprint/_help.py

```diff
@@ -0,0 +1,452 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['blueprint'] = """
+    type: group
+    short-summary: Commands to manage blueprint.
+"""
+
+helps['blueprint create'] = """
+    type: command
+    short-summary: Create a blueprint definition.
+    examples:
+      - name: Create a subscription blueprint
+        text: |-
+               az blueprint create --subscription MySubscription --name \\
+               MyBlueprint --description \\
+               "blueprint contains all artifacts" --target-scope \\
+               subscription
+      - name: Create a management group blueprint
+        text: |-
+               az blueprint create --management-group MyManagementGroup --name \\
+               MyBlueprint --description \\
+               "blueprint contains all artifact" --target-scope \\
+               subscription
+"""
+
+helps['blueprint update'] = """
+    type: command
+    short-summary: Update a blueprint definition.
+"""
+
+helps['blueprint delete'] = """
+    type: command
+    short-summary: Delete a blueprint definition.
+    examples:
+      - name: Delete a management group blueprint
+        text: |-
+               az blueprint delete --management-group MyManagementGroup --name \\
+               MyBlueprint
+      - name: Delete a subscription blueprint
+        text: |-
+               az blueprint delete --subscription MySubscription --name \\
+               MyBlueprint

```

Comment:
For blueprint name, is it be unique in the world? 
The combination of scope info and name is to get the blueprint id, right? In this way, can we allow blueprint id input and add a custom validator if parse_resource_id doesn't work for your scenario?

Reply:
We can use some patch to parse_resource_id to identify management group blueprint id. But we may need further patch for artifact id, published blueprint id, published blueprint artifact id, blueprint assignment id. It's not worth the extra effort for breadth coverage. Let's wait for msrestazure fix. And also Blueprint is not GAed and may be replaced by a new RP. Let's just roll it out quickly and add convenient feature if we need to go deeper. I appreciate your suggestions and it definitely make sense in the long run.